stat.go

```diff
@@ -142,14 +142,20 @@ func parseSoftIRQStat(line string) (SoftIRQStat, uint64, error) {
 	return softIRQStat, total, nil
 }
 
+func newStat() Stat {
+	return Stat{
+		CPU: make(map[int64]*CPUStat),
+	}
+}
+
 // NewStat returns information about current cpu/process statistics.
 // See https://www.kernel.org/doc/Documentation/filesystems/proc.txt
 //
 // Deprecated: Use fs.Stat() instead.
 func NewStat() (Stat, error) {
 	fs, err := NewFS(fs.DefaultProcMountPoint)
 	if err != nil {
-		return Stat{}, err
+		return newStat(), err
```

Comment:
Since we've changed the CPUs into a map, we have to initialzie it. Return Stat{} will initialize the map in nil, which burdens initialization in the usage site.

Reply:
I think just returning Stat{} with map being nil is fine. If an error is returned, the caller shouldn't use it anyway, right?

Reply:
I agree with @discordianfish, I think just returning an empty Stat{} with nil map should be fine.

Reply:
OK.